cliquet/logs.py

```diff
@@ -104,7 +104,7 @@ def __call__(self, logger, name, event_dict):
         return log_msg
 
 
-class MozillaHekaRenderer(structlog.processors.JSONRenderer):
+class MozillaHekaRenderer(object):
```

Comment:
Not inherit from default JSON renderer because ujsons.dumps() does not support cls kwarg
https://github.com/hynek/structlog/blob/master/structlog/processors.py#L146